rules/config

```diff
@@ -131,7 +131,7 @@ INCLUDE_MGMT_FRAMEWORK = y
 
 # ENABLE_HOST_SERVICE_ON_START - enable sonic-host-server for mgmt-framework and/or
 # telemetry containers to access host functionality by default
-ENABLE_HOST_SERVICE_ON_START = n
+ENABLE_HOST_SERVICE_ON_START = y
```

Comment:
>y [](http://example.com/codeflow?start=31&length=1)

Could you keep `n` ? Or do you enforce the config?

Reply:
This "make" flag does not correspond to a configurable attribute. Because this feature adds an active user of the D-Bus mechanism provided by the sonic-hostservices process, the "make" flag should now be enabled by default. 

Otherwise, the user would need to manually start the service to enable the feature. (If the default state is "off", the default behavior of the new feature would be to fail when the command is invoked.)

Reply:
Will this increase the fast-reboot/warm-reboot time? If you think this is not confiurable, you should remove it in this file.

Reply:
@jleveque @yxieca @lguohan Please help review.

Reply:
@lguohan - Please update so that we can take this to closure.

Reply:
@lguohan - Did you get a chance to look at this one ?

Reply:
I have reverted the change discussed in this thread in order to resolve this conversation.

Reply:
Thx Kerry. @qiluo-msft - Can you please approve this PR ?